Octokit/Http/HttpClientAdapter.cs

```diff
@@ -109,6 +103,8 @@ protected virtual HttpRequestMessage BuildRequestMessage(IRequest request)
             {
                 var fullUri = new Uri(request.BaseAddress, request.Endpoint);
                 requestMessage = new HttpRequestMessage(request.Method, fullUri);
+
+                requestMessage.Properties["AllowAutoRedirect"] = request.AllowAutoRedirect;
```

Comment:
It was one of the first things I'd removed when refactoring earlier attempts. Good enough for me :fire::fire::fire::fire::fire: